www/tutorial/README.md

```diff
@@ -1,61 +1,45 @@
-do the things they needed to do. It's name, in fact, was `command_simulator.rb`, and it lived in a
-nearby directory named `inspec/tasks`. And, aha, a match was found.
+do the things they needed to do. It's name, in fact, was `run_simulator_recording.rb`, and it lived in a nearby directory named `scripts`. And, aha, a match was found.
 
-And so command_simulator kindly took tutorialyml by the hand, and said 'let me help you! I can parse you, and help you help others'.
+And so `run_simulator_recording.rb` kindly took `tutorial.yml` by the hand, and said 'let me help you! I can parse you, and help you help others'.
 And thus it began...
-command_simulator parsed tutorialyml to find the instructions and commands that are noted in the instructions.
-After that, the real fun began: command_simulator worked and worked to format the commands and prepare them.
-She created json files for instructions and commands, and used Train with MixLib:Shellout in the background to
-run the commands and record the output to some .txt files in the `www/tutorial/app/responses` directory.
-Pleased with her work, command_simulator turned to her friend the webapp, and said to her: 'take these json files, and show them to the people.'
+`run_simulator_recording.rb` parsed `tutorial.yml` to find the instructions and commands that are noted in the instructions. After that, the real fun began: `run_simulator_recording.rb` worked and worked to format the commands and prepare them.
+She created `.json` files for instructions and commands, and used [Train](https://github.com/chef/train) in the background to run the commands and record the output to some .txt files in the `app/responses` directory.
+Pleased with her work, `run_simulator_recording.rb` turned to her friend the webapp, and said to her: 'take these json files, and show them to the people.'
 
 Webapp then went and found her friend xterm, and said, why let's do this together, for two heads are always better than one!
 And so webapp said to her dear friend angular2, please help me make this work! and angular2 came running, as friends always do.
 And so angular2 met xterm, and as they shook hands, they knew they would be good friends together. So they put their heads together,
-and made some decisions about what each one's role would be. Xterm claimed responsiblity for creating the terminal view,
+and made some decisions about what each one's role would be. Xterm claimed responsibility for creating the terminal view,
 reading the user's input, and displaying it.  App then explained she could take that final user input sent over by xterm and
-and transform it into a regular expression, which she could then use to match against the information in the commandsjson file,
+and transform it into a regular expression, which she could then use to match against the information in the `commands.json` file,
 matching the user's input against the key and then retrieving the associated value's txt file.
 And it was so. :)
 
```

Comment:
I was not sure, I liked the idea somehow, but we could shorten that up a bit :-)